generate/seccomp/consts.go

```diff
@@ -0,0 +1,7 @@
+package seccomp
+
+const (
+	overwrite = "overwrite"
+	appnd     = "append"
```

Comment:
Well, the only issue is that it would then be exported, not sure if that makes a difference